src/Classes/Options/Particles/Rotate.ts

```diff
@@ -0,0 +1,33 @@
+import {IRotate} from "../../../Interfaces/Options/Particles/IRotate";
+import {IRotateAnimation} from "../../../Interfaces/Options/Particles/IRotateAnimation";
+import {Utils} from "../../Utils/Utils";
+import {RotateAnimation} from "./RotateAnimation";
+import {RotateDirection} from "../../../Enums/RotateDirection";
+
+export class Rotate implements IRotate {
+    public animation: IRotateAnimation;
+    public direction: RotateDirection;
+    public random: boolean;
+    public value: number;
+
+    constructor() {
+        this.animation = new RotateAnimation();
+        this.direction = RotateDirection.clockwise;
+        this.random = false;
+        this.value = 0
+    }
+
+    public load(data: IRotate): void {
+        if (Utils.hasData(data)) {
+            this.animation.load(data.animation);
+
+            if (Utils.hasData(data.random)) {
+                this.random = data.random
+            }
+
+            if (Utils.hasData(data.random)) {
+                this.value = data.value
+            }
+        }
+    }
+}
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [file should end with a newline](https://app.codacy.com/manual/ar3s/tsparticles/pullRequest?prid=5164124)